src/storage/src/hummock/sstable/forward_sstable_iterator.rs

```diff
@@ -125,15 +127,16 @@ impl SstableIterator {
             return Ok(());
         }
         // Maybe the previous preload stream breaks on some cached block, so here we can try to preload some data again
-        if self.preload_stream.is_none()
-            && idx + 1 < self.preload_end_block_idx
-            && let Ok(preload_stream) = self
-                .sstable_store
-                .preload_blocks(self.sst.value(), idx, self.preload_end_block_idx)
-                .await
+        if self.preload_stream.is_none() && idx + 1 < self.preload_end_block_idx && let Ok(preload_stream) = self
+            .sstable_store
+            .prefetch_blocks(self.sst.value(), idx, self.preload_end_block_idx,
+                        self.options.cache_policy,
+                        &mut self.stats,
+            ).await
```

Comment:
Not related to this PR but I just realize that the error returned by `prefetch_blocks` is ignored. We should check and log it.

Reply:
Fixed